### PR TITLE
Appending Remote word in working set

### LINKED ISSUE #14309: [Remote File] If user save the file name having same name as url then nothing is displayed in front of file name.
### Description
If user save the file name having same name as url then nothing is displayed in front of file name.

### Steps to Reproduce
1. Launch brackets 1.13.
2. Press ctrl+shift+O for win and cmd+shift+O for mac.
3. Quick open modal bar will open.
4. Enter a network url that supports http or https. Example: http://www.muse-php.com
5. The code for the page is displayed in brackets and file is visible in working files in file tree.
6. Go to file -> Save As
7. Save the file with same name i.e. muse-php.com

two file

**Expected behavior:** Their should be a text so that user can differentiate between two files.

**Actual behavior:** If user save the file name having same name as url then nothing is displayed in front of file name.

### Versions
Win 10 64 Bit
Mac 10.12
Release 1.13 build 1.13.0-17668

### PR DESCRIPTION
**Pull request by navch**
Fixes #14309

### CODE REVIEW

File: src/utils/ViewUtils.js [hunk 1]
```diff
@@ -427,6 +428,15 @@ define(function (require, exports, module) {
             var fp = file.fullPath.split("/");
             fp.pop(); // Remove the filename itself
             displayPaths[index] = fp.pop();
+
```
Comment (shubhsnov) [inline src/utils/ViewUtils.js]: See if we can use useIconProviders and useClassProviders from Working Set View for this as well

Comment (shubhsnov) [reply src/utils/ViewUtils.js]: ping @swmitra  for guidance. I believe the "Remote" word can be added to the :after of a class and then that class can be registered with working set view.

**Comment by swmitra** [comment]
We should use the file tree decoration model by providing icon and class provider. It should be a pure CSS solution in decorating the remote file entries.

**Review by swmitra — CHANGES_REQUESTED** [review]
Let's use just the protocol name as marker. It's more friendly for a developer instead of calling Remote. This can be done using a pseudo selector with content set to protocol name.

**Comment by navch** [comment]
@swmitra Review comment addressed. Please have a look now.

**Comment by shubhsnov** [comment]
@navch  I think we should add prototype.name in all File, InMemoryFile and RemoteFile and then check against that directly

**Comment by boopeshmahendran** [comment]
Could we check `constructor.name` like [this](https://github.com/adobe/brackets/blob/master/src/extensions/default/NavigationAndHistory/main.js#L604)?

**Comment by swmitra** [comment]
Closing this PR as #14330 adds protocol badge to all remote file entries in working set and recent file list.

**Pull request closed without merging**